run-tests.sh

```diff
@@ -238,6 +238,12 @@ if ! (g3kubectl get pods --no-headers -l app=guppy | grep guppy) > /dev/null 2>&
   donot '@guppyAPI'
 fi
 
+
+if ! jq -re '.|values|map(select(.data_file_properties.consent_codes!=null))|.[]' < "$TEST_DATA_PATH/schema.json" > /dev/null; then
```

Comment:
I think if `TEST_DATA_PATH` is not set then a lot of stuff will be broken--see `jenkins-simulate-data.sh` and `utils/nodes.js` 
But it will look in root for schema.json so... it probably would fail 🤷‍♂

Reply:
definitely! i'll try to explain my question: at other places in the code, if `TEST_DATA_PATH` is not set, we get a nice error message. if this line crashes when it's not set, the pipeline will crash before we get to the nice error messages and we won't know what happened, because i don't think the crash would mention that `TEST_DATA_PATH` is not set. it would probably be something like `cannot find schema.json`? would you mind checking? and if it does crash, you can maybe change the logic to disable `@indexRecordConsentCodes` if `TEST_DATA_PATH` is not set

Reply:
👍 👍 I'll just exit 1--seems like a bad idea to default to _disabling_ tests because of a tangential config issue, not to mention it would crash sooner or later anyway